src/chainparams.cpp

```diff
@@ -162,7 +162,10 @@ class CMainParams : public CChainParams {
 
         // Note that of those with the service bits flag, most only support a subset of possible options
         vSeeds.push_back(CDNSSeedData("multidoge.org", "seed.multidoge.org", true));
-        vSeeds.push_back(CDNSSeedData("multidoge.org", "seed2.multidoge.org"));
+        vSeeds.push_back(CDNSSeedData("multidoge.org", "seed2.multidoge.org")); // Seeders are dead
```

Comment:
according to my checks over the past 24h, this seeder is not dead. The community may want to put some downtime detection on individual seeders though.